src/main/views/user-deletions/template.njk

```diff
@@ -44,7 +44,12 @@
         classes: "govuk-!-width-three-quarters govuk-!-margin-top-1",
         id: "emailAddress",
         name: "emailAddress",
-        value: deletedUsersForm.emailAddress
+        value: deletedUsersForm.emailAddress,
+        type: "email",
+        spellcheck: false,
+        hint: {
+          text: "We’ll use this to search the users"
```

Comment:
Do we need hint?

Reply:
as per thus https://design-system.service.gov.uk/patterns/email-addresses/